hooks/README.md

```diff
@@ -8,7 +8,7 @@ Enable web developers to differentially deliver lightweight, fast experiences fo
 Now that the adaptive React hooks are [custom hooks](https://reactjs.org/docs/hooks-custom.html) we can just use them like normal React hooks after importing them.
 
```

Comment:
The changes I wanted to review first were to the README - e.g were we able to make it easier to use the Hooks. I think you did a great job here reducing what needs to be imported to exactly the right level for developers to quickly understand these. Nice work :)

I'll review the implementation in more detail through the day.

Reply:
FYI: hooks/README is also updated.